docs/how_to/custom_plugins.rst

```diff
@@ -265,7 +265,7 @@ Now we need to change our plugin definition to use this model, so our new
         cache = False
 
         def render(self, context, instance, placeholder):
-            context['instance'] = instance
+            context = super(HelloPlugin, self).render(context, instance, placeholder)
```

Comment:
the context passed as parameter does not have instance / placeholder: they are added in the parent render method